packages/components/src/Tabs/Tabs.tsx

```diff
@@ -78,6 +78,12 @@ export function Tabs({ children, defaultTab = 0, onTabChange }: TabsProps) {
     };
   });
 
+  useEffect(() => {
```

Comment:
Is there a better approach than this? Will this mess up any other scenarios?

Reply:
I didn't run or add unit tests yet. I can do that once we are aligned on the solution or guidance. Thanks!